math/vecops/inc/ROOT/RVec.hxx

```diff
@@ -998,6 +998,8 @@ void swap(RVec<T> &lhs, RVec<T> &rhs)
 /// auto sortIndices = Argsort(v);
 /// sortIndices
 /// // (ROOT::VecOps::RVec<unsigned long> &) { 2, 0, 1 }
+/// Take(v, sortIndices)
+/// // (ROOT::VecOps::RVec<double> &) { 1., 2., 3. }
```

Comment:
```suggestion
/// (ROOT::VecOps::RVec<double> &) { 1., 2., 3. }
```
why the extra `//`?

Reply:
It's then a comment in the code block and not actual code!

Reply:
You can have a look here how it looks: https://root.cern/doc/master/namespaceROOT_1_1VecOps.html#a7dcd060b97f6c82621ba0d8f376ad195

Reply:
ah I see that's the style used for everything except https://root.cern/doc/master/namespaceROOT_1_1VecOps.html#abbbf82257156b750c7b9bdd39a53ee33 , ok then